cypress/integration/app.spec.ts

```diff
@@ -98,6 +98,7 @@ describe('App', () => {
 
       cy.findByRole('heading', { name: 'nexus_entry / image' }).should('exist');
       cy.findByRole('tab', { name: 'NX Image' }).should('exist');
+      cy.wait(500);
```

Comment:
:roll_eyes: 

Sometimes this test makes the following test pending. I suspect it is because the texture computation in the worker didn't finish. Adding this wait makes it consistently passing